prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirTransformer.kt

```diff
@@ -36,23 +33,6 @@ class FhirTransformer(
     private val valueXRegex = Regex("""value[A-Z][a-z]*""")
```

Comment:
Unrelated to this PR, should FhirTransformer be a part of the prime-fhir-converter library? I can see the argument either way but I don't know if there has been any discussion.

Reply:
I started tracking that info on a new tab in the sheet victor started: https://docs.google.com/spreadsheets/d/116ju7q4YtTxgRxn-v1hOwVcsEq4m0wPxufjXeEYP_kY/edit?usp=sharing